src/RouterSelector.web.js

```diff
@@ -83,7 +83,7 @@ const handleLinks = async () => {
 
 let SignupRouter = React.lazy(async () => {
   const [module] = await Promise.all([
-    retryImport(() => import(/* webpackChunkName: "signuprouter" */ './SignupRouter')),
+    retryImport(() => import(/* webpackChunkName: "signuprouter" */ './routers/SignupRouter')),
```

Comment:
Should we change these imports in RouterSelector.native.js too?

Reply:
with latest changes I reverted changes in routes. fixed.